### PR TITLE
Feature/enable feature toggeling

### PR DESCRIPTION
**Pull request by manuwei**
* enable feature toggling
* implement a UI to toggle the feature (only shows the status of the feature toggles when showSetting is set to false) 
+ implement an read-only mode if the feature toggle "editing" is set to false

### CODE REVIEW

File: .docker/feature.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "features": {
+    "designModel": false,
+    "patternCandidate": false,
+    "patternViews": false,
+    "issue": false
```
[the remark this answers is not in the document]
Comment (salmma) [reply .docker/feature.json:6]: please fix the automated checks :)

**Comment by manuwei** [comment]
Thanks for the comments @salmma, I addressed them in the previous commits

File: src/assets/env.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+(function (window) {
+  window['env'] = window['env'] || {};
+
+  // Environment variables
+
+  window['env']['production'] = false;
+  window['env']['CONFIG_SERVER_HOST_NAME'] = 'localhost';
+  window['env']['QC_ATLAS_HOST_NAME'] = 'localhost';
+  window['env']['NISQ_ANALYZER_HOST_NAME'] = 'localhost';
```
Comment (salmma) [inline src/assets/env.js:9]: this can also be removed, I guess

**Review by salmma — APPROVED** [review]
LGTM

**Pull request merged**